### PR TITLE
Blog App: Creating data model

### PR DESCRIPTION
**Pull request by arnoldnekemiah**
- Created a new app. And gave it a meaningful name.
- Initialized your project with Git.
- Made sure that the project has Postgres database set up.

Built project schema;

-   Created and run the necessary migration files.
-   Table and column names should the ERD diagram.
    photo for users table should be a link to an image
-   Foreign keys keys are included.
-   Paid attention that in posts table the column holding the foreign key to the users table should be named author_id
-   All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.bigint "author_id", null: false
+    t.string "title"
+    t.text "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_posts_on_author_id"
+  end
```
Comment (julie-ify) [inline db/schema.rb]: - [x] The posts table should have likes and comments counters with integer column types. Kindly run new migrations and add these columns.

  ```
  t.integer "likes_counter"
  t.integer "comments_counter"
  ```

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "photo"
+    t.text "bio"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["name"], name: "index_users_on_name"
+  end
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] The users' table should have the counter of a post with an integer column type. Kindly run new migrations and add this column.

  ```
  t.integer "posts_counter"
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @arnoldnekemiah,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
```
Comment (cynthiainga) [inline db/schema.rb]: - [ ] As mentioned by the previous reviewer the users' table should have the counter of a post with an integer column type. Kindly add the posts_counter column 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @arnoldnekemiah, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Descriptive PR ✔️ 

 ## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇🏾 👇🏾 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @arnoldnekemiah, 
congrats 2

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**